examples/notebooks/get-started/try-apache-beam-yaml.ipynb

```diff
@@ -0,0 +1,424 @@
+    "pipeline:\n",
+    "  type: chain\n",
+    "\n",
+    "  transforms:\n",
+    "    # Read input data. Each line from the csv file is a String.\n",
+    "    - type: ReadFromText\n",
+    "      name: SmsData\n",
+    "      file_pattern: data/SMSSpamCollection.csv\n",
+    "\n",
+    "    # Split each line into an array, where the first element is message label (ham or spam) and the second is the content of the message.\n",
+    "    - type: PyMap\n",
+    "      name: SplitLine\n",
+    "      fn: 'lambda line: line.split(\"\\\\t\")'\n",
+    "\n",
+    "    # Select only the rows that contain spam messages, based on the label.\n",
+    "    - type: PyFilter\n",
+    "      name: SpamMessages\n",
+    "      keep: 'lambda row: row[0] == \"spam\"'\n",
+    "\n",
+    "    # Using a regex, we'll split the content of the message (one long string) into words (list of strings)\n",
+    "    - type: PyFlatMap\n",
+    "      name: FindWords\n",
+    "      fn: |\n",
+    "        import re\n",
+    "        lambda line: re.findall(r\"[a-zA-Z]+\", line[1])\n",
+    "\n",
+    "    # Transforming each word to lower case and combining it with a '1'. Result of this step are pairs (word: 1).\n",
+    "    - type: PyMap\n",
+    "      name: PairLoweredWordsWith1\n",
+    "      fn: 'lambda word: (word.lower(), 1)'\n",
+    "\n",
+    "    # Using SumPerKey transform, we'll calculate the occurence of each word.\n",
+    "    - type: SumPerKey\n",
+    "      name: GroupAndSum\n",
+    "\n",
+    "    # Select 10 most popular words. Input format to this step is a tuple (word: count),\n",
+    "    # so we provide the count (row[1]) as the key to compare the numbers\n",
+    "    - type: TopNLargest\n",
+    "      name: Largest\n",
+    "      n: 10\n",
+    "      key: 'lambda row: row[1]'\n",
+    "\n",
+    "    # Save results to a text file.\n",
+    "    - type: WriteToText\n",
+    "      name: SaveToText\n",
+    "      file_path_prefix: \"data/result-pipeline-02\"\n",
+    "      file_name_suffix: \".txt\"\n",
+    "'''\n",
+    "save_to_file(pipeline, 'pipeline-02.yaml')"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Let's run the pipeline:"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "run('python -m apache_beam.yaml.main --pipeline_spec_file=pipeline-02.yaml')"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "To view the output:"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "run('head data/result-pipeline-02-00000-of-00001.txt')"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Summary\n",
```

Comment:
If it's not too much extra work, it may be good to include a WordCount example so that folks can compare word count of YAML vs different SDKS (the other SDK quick starts use word count).